src/main/resources/data/computercraft/lua/rom/apis/colors.lua

```diff
@@ -8,6 +8,127 @@
 -- and it will use the other API, colours which is exactly the same, except in
 -- British English (e.g. @{colors.gray} is spelt @{colours.grey}).
 --
+-- On basic terminals (such as the Computer and Monitor), all the colors are
```

Comment:
Might be worth converting this entire thing to be a comment in the style of this?

```lua
--[-

]]
```

Really any non-trivial comment should be like this, but I've never been consistent.